### PR TITLE
Branched Away from Sleek

### PR DESCRIPTION
**Pull request by CharlieS1103**
The creator and I spoke and they agreed that I should branch off as well since we're heading in two different directions.
I plan on making Dreary theme(nice name amiright,) have more features and things to look at than the sleek theme, whereas he would like to keep Sleek minimalist. 

I do plan on keeping it fairly minimalist but adding borders and things of that sort to break it up.
I also will most likely move elements around and round some of them out, and create a multitude of more schemes. 
I will use media queries to change the playback icon color for each scheme and will make the theme more reactive in general. 
And of course whatever wild ideas I think of while working on it(as seen by my ridiculous commit organization, I seem to have no trouble with that.)

### CODE REVIEW

**Comment by morpheusthewhite** [comment]
I'm fine with branching off the theme if it is different. But as I said elsewhere I don't really like some elements of the UI. More specifically
1. Borders around playlist names 
image
The rest of the unmodified UI has no border. This is not really coherent.
2. Borders around friends
image
In this case, if you have one friend it is not evident, but when there are many it adds a lot of foreground elements to the UI, making it much more full of stuff to focus on
3. It is hard to read some text, e.g. 
image

P.S.: Please fix your history, now there are a lot of commits. [Squash them](https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git).

**Comment by CharlieS1103** [comment]
@morpheusthewhite I just changed a ton of things, tell me what you think! Btw, I'm not quite sure how to Squash them, as you can see I tried to squash them in the "Changes" commit (id: 6d29774 )

**Comment by CharlieS1103** [comment]
Btw sorry about not being the best at git, I'll be sure to watch a few videos on it over the weekend.

File: Dreary/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+![Deeper Screenshot](deeper.png)
+
+### Lover
+![Lover Screenshot](lover.png)
+
+### RainforestCafe
+![Rainforest Cafe Screenshot](rainforestcafe.png)
+
+##Important
+Sidebar Config option must be enabled for theme to work properly, you can use it without, however some features may not be available.
```
Comment (morpheusthewhite) [inline Dreary/README.md:24]: What are you referring to?

File: Sleek/README.md [hunk 2]
```diff
@@ -11,8 +11,6 @@
 ### Psycho
 ![Psycho Screenshot](psycho.png)
 
-### Deeper
```
Comment (morpheusthewhite) [inline Sleek/README.md:14]: Why did you remove the colour scheme? Please add it again, I cannot select which commits are merged in a PR

File: Sleek/color.ini [hunk 3]
```diff
@@ -19,27 +19,6 @@ notification-error = 5e0000
 playback-bar       = ff4700
 misc               = adadad
 
-[Deeper]
```
Comment (morpheusthewhite) [inline Sleek/color.ini:22]: Again, add the colour scheme to Sleek

**Review by morpheusthewhite — CHANGES_REQUESTED** [review]
Restore Sleek theme as before

**Comment by morpheusthewhite** [comment]
> @morpheusthewhite I just changed a ton of things, tell me what you think! Btw, I'm not quite sure how to Squash them, as you can see I tried to squash them in the "Changes" commit (id: [6d29774](https://github.com/morpheusthewhite/spicetify-themes/commit/6d297749a39cc8bbb17fc7f77f8fb86bd6c13ab1) ).
 
Yes, It didn't work. I would suggest leaving this PR open until the theme is ready, then open another one with just one commit (since other people are also pushing on this repository, squashing may be a mess).

I generally like much more the theme now. But I still see some problems:
1. There are still some texts which are hard to read, especially in the playlist cards, e.g. 



> Also sorry about changing the sleek theme, I just assumed the original owner would want things reverted! You can completely ignore the file changes involving sleek.

I cannot Ignore it, please add it again.

**Comment by CharlieS1103** [comment]
@morpheusthewhite Alright I finished it.

File: Dreary/README.md (hunk 1 above)
Comment (CharlieS1103) [reply Dreary/README.md:24]: @morpheusthewhite I'm referring to the sidebar changes, such as the border around playlists, I can see about making it work without but I figured that for now its fine.

**Comment by morpheusthewhite** [comment]
> I generally like the theme much more now. But I still see some problems:
> 
> 1. There are still some texts which are hard to read, especially in the playlist cards, e.g.
>        image
>        image
> 
> 2. I find too much inner padding in the playlist names. It would be better to make it visually more consistent with the playlist cards, e.g.
>        image
>        image
> 
> 3. I generally think you should improve the colour schemes, especially the 2 new ones. Read [this post](https://www.smashingmagazine.com/2010/02/color-theory-for-designer-part-3-creating-your-own-color-palettes/), it may help you a lot.

Please work also on this.

**Comment by CharlieS1103** [comment]
@morpheusthewhite I followed your suggestions, I will clean up the rest of the new themes in a bit.

**Comment by CharlieS1103** [comment]
yeah as you can  see i attempted to squash my commits, did not work well.
